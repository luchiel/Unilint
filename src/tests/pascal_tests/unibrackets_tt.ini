[spaces_and_newlines]
spaces_inside_braces = true
spaces_inside_brackets = true
;spaces_inside_curly_braces makes no sence for pascal