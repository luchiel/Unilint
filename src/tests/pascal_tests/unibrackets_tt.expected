2:13: no space after [
2:16: no space before ]
5:7: no space after (
5:9: no space after [
5:9: no space before ]
5:16: no space after [
5:16: no space before ]
5:17: no space before )
